modules/test/playwright/tests/frontend-data-set-views-web/visualizationModes.spec.ts

```diff
@@ -284,12 +266,19 @@ test.describe('Visualization Modes in Data Set Manager', () => {
 		await test.step('Add fields', async () => {
 			await visualizationModesPage.openAddFieldsModal();
 
-			await visualizationModesPage.addRootField(SAMPLE_SCALAR_FIELD);
-			await visualizationModesPage.addRootField(SAMPLE_OBJECT_FIELD);
-			await visualizationModesPage.addChildField(
-				[SAMPLE_OBJECT_FIELD],
-				SAMPLE_OBJECT_CHILD_FIELD
-			);
+			await visualizationModesPage.selectField({
+				fieldName: SAMPLE_SCALAR_FIELD,
+			});
+
+			await visualizationModesPage.selectField({
+				dataId: `${SAMPLE_OBJECT_FIELD}.*`,
+				fieldName: SAMPLE_OBJECT_FIELD,
+			});
```

Comment:
Hardcoding is the simplest solution for complex fields delimiters here. If there is a need, if we'll have multiple tests for complex fields, we can improve this later on.